runtime/src/accounts_index.rs

```diff
@@ -764,7 +773,7 @@ impl<T: IsCached> AccountsIndex<T> {
         Self::new(Some(BINS_FOR_TESTING))
     }
 
-    pub fn new(bins: Option<usize>) -> Self {
+    pub fn new(bins: Option<AccountsIndexConfig>) -> Self {
```

Comment:
Similar to the other comment, maybe the `bins` parameter should be `accounts_index_config`.